integration/test_flow_integration.py

```diff
@@ -117,16 +119,67 @@ def _get_redis_kv_all_attrs(setup_teardown_test: ContextForTests, key: str):
     }
 
 
-def get_key_all_attrs_test_helper(setup_teardown_test: ContextForTests, key: str):
+def _get_sql_by_key_all_attrs(
+    setup_teardown_test: ContextForTests, key: Union[str, List[str]], key_name: Union[str, List[str]]
+):
+    import sqlalchemy as db
+
+    where_statement = ""
+    if isinstance(key, str):
+        key = key.split(".")
+    if isinstance(key, list):
+        for i in range(len(key_name)):
+            if i != 0:
+                where_statement += " and "
+            where_statement += f'{key_name[i]}="{key[i]}"'
```

Comment:
As discussed, we should only wrap values with quotes if the type is string. Not sure if we have test coverage of this.